docs/content/Coding Milestones/PoC2023q1/example1-subs/example1-stage-2.md

```diff
@@ -218,6 +220,7 @@ to the httpd workload but is here to demonstrate that `APIService`
 objects can be downsynced.
 
 ```shell
+# TODO: apiregistration.k8s.io is not supported in kubeflex
```

Comment:
It's apiservices   not apiregistration.k8s.io  , and it's not KubeFlex it's an issue we had with dynamic informer and a fix was already merged. 
Maybe change the comment to "TODO: verify that we APIservice is supported for KF Control Plan".  If it's not we will remove  APIservice  from the example. This is used only for KCP.

Reply:
done